share/OpenMS/examples/TOPPAS/Ecoli_Identification.toppas

```diff
@@ -140,7 +140,7 @@
     <NODE name="2" description="">
       <ITEM name="recycle_output" value="false" type="string" description="" required="false" advanced="false" />
       <ITEM name="toppas_type" value="tool" type="string" description="" required="false" advanced="false" />
-      <ITEM name="tool_name" value="OMSSAAdapter" type="string" description="" required="false" advanced="false" />
+      <ITEM name="tool_name" value="TODO" type="string" description="" required="false" advanced="false" />

```

Comment:
Could we add CometAdapter here, or does the toppas workflow would have to be redone?

Reply:
needs to be redone

Reply:
will do now